templates/circle.yml.erb

```diff
@@ -1,3 +1,9 @@
+checkout:
+  post:
+    - cp .sample.env .env
+database:
+  override:
+    - bundle exec rake db:setup
```

Comment:
> In the travis version of the configuration we actually ran bin/setup which does more than required for running tests but is a useful test of the new developer/designer onboarding workflow. See discussion here: #441

@pedrosmmoreira what do you think of running `bin/setup` in the section `database.override`?